### ISSUE TITLE
[blackhole] font size too big for desktop numbers

### ISSUE THREAD

**Issue by ploum** [regolith-linux/regolith-look-extra#2]
Since last update, for whatever reason, font is way bigger in the bar for the "desktop" numbers.

See screenshot. On the left, the desktop numbers (too big) and, on the right, i3xrocks applets with the correct font size.

I have no idea what’s happening here. Maybe blackhole should be moved to the default font to avoid those problems.

![Capture d’écran du 2022-04-29 10-58-48](https://user-images.githubusercontent.com/1233155/165915079-3cb2c662-0bb2-49bb-b26b-a4e088f124f0.png)

**Comment by ploum** [comment]
The last update doesn’t solve the issue (the font of the bar was made bigger to somewhat match the font of the workspace but the font of the workspace seems still not modifiable).

Afaik, the main issue seems to be that workspace switcher has a non-configurable font-size  (I’ve tried every font setting and it stay unaffected)

**Comment by ploum** [comment]
I’m wondering what line 

#define glyph_font QUOTE(typeface_bar_glyph)   (in i3-wm) 

is supposed to do. The "typeface_bar_glyph" variable seems to not be set

**Comment by kgilmer** [comment]
Yes, you're right, the `typeface_bar_glyph` was removed in the recent refactor.  It was replaced correctly in the default look but I missed it in the extras.  See referenced commit.  I also reduced the font size from 11 to 9.
